test/images/image-util.sh

```diff
@@ -49,57 +75,78 @@ build() {
-    fi
-
-    # copy the qemu-*-static binary to docker image to build the multi architecture image on x86 platform
-    if grep -q "CROSS_BUILD_" Dockerfile; then
-      if [[ "${arch}" == "amd64" ]]; then
-        ${SED} -i "/CROSS_BUILD_/d" Dockerfile
-      else
-        ${SED} -i "s|QEMUARCH|${QEMUARCHS[$arch]}|g" Dockerfile
-        # Register qemu-*-static for all supported processors except the current one
-        echo "Registering qemu-*-static binaries in the kernel"
-        local sudo=""
-        if [[ $(id -u) != 0 ]]; then
-          sudo=sudo
+  case "$(uname -m)" in
+    x86_64|amd64)
+      for arch in ${archs}; do
+        prepareBuild "${arch}"
+        # image tag
+        TAG=$(<VERSION)
+        # copy the qemu-*-static binary to docker image to build the multi architecture image on x86 platform
+        if grep -q "CROSS_BUILD_" Dockerfile; then
+          if [[ "${arch}" == "amd64" ]]; then
+            ${SED} -i "/CROSS_BUILD_/d" Dockerfile
+          else
+            ${SED} -i "s|QEMUARCH|${QEMUARCHS[$arch]}|g" Dockerfile
+            # Register qemu-*-static for all supported processors except the current one
+            echo "Registering qemu-*-static binaries in the kernel"
+            local sudo=""
+            if [[ $(id -u) != 0 ]]; then
+              sudo=sudo
+            fi
+            ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset
+            curl -sSL https://github.com/multiarch/qemu-user-static/releases/download/"${QEMUVERSION}"/x86_64_qemu-"${QEMUARCHS[$arch]}"-static.tar.gz | tar -xz -C "${temp_dir}"
+            # Ensure we don't get surprised by umask settings
+            chmod 0755 "${temp_dir}/qemu-${QEMUARCHS[$arch]}-static"
+            ${SED} -i "s/CROSS_BUILD_//g" Dockerfile
+          fi
         fi
-        ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset
-        curl -sSL https://github.com/multiarch/qemu-user-static/releases/download/"${QEMUVERSION}"/x86_64_qemu-"${QEMUARCHS[$arch]}"-static.tar.gz | tar -xz -C "${temp_dir}"
-        # Ensure we don't get surprised by umask settings
-        chmod 0755 "${temp_dir}/qemu-${QEMUARCHS[$arch]}-static"
-        ${SED} -i "s/CROSS_BUILD_//g" Dockerfile
-      fi
-    fi
-
-    docker build --pull -t "${REGISTRY}/${IMAGE}-${arch}:${TAG}" .
-
-    popd
-  done
+        docker build --pull -t "${REGISTRY}/${IMAGE}-${arch}:${TAG}" .
+        popd
+      done
+      ;;
+    arm*|aarch*)
+      for arch in ${archs}; do
+        if [[ $arch == arm ]] || [[ $arch == arm64 ]]; then
+          prepareBuild "${arch}"
+          # Cross build on non-amd server is not supported, build native image only
+          ${SED} -i "/CROSS_BUILD_/d" Dockerfile
+          # image tag
+          TAG=$(<VERSION)
+          docker build --pull -t "${REGISTRY}/${IMAGE}-${arch}:${TAG}" .
+          popd
+        fi
+      done
+      ;;
+    powerpc*|ppc*)
```

Comment:
just `ppc64le` is good enough.

Reply:
possible value is  `ppc`, `ppc64`, `ppc64le` I think

Reply:
well, docker is supported only on `ppc64le` value, rest all unsupported.

Reply:
thanks, I changed this to match with ppc*.

Reply:
update it again to exactly match with `ppc64le`.